Client/Application/WebServer.swift

```diff
@@ -18,8 +18,10 @@ class WebServer {
     }
 
     func start() -> Bool {
-        if !server.running {
-            try! server.startWithOptions([GCDWebServerOption_Port: 6571, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true])
+        withExtendedLifetime(server) {
```

Comment:
This was the same fear I had when skimming this PR: that this 'distracts' ARC rather than fixing the bug.

If `server` is somehow being deinited despite being strongly held by a singleton, then something even more scary is happening.